dataland-frontend/src/components/general/SlideShow.vue

```diff
@@ -37,19 +37,32 @@ const slider = ref<HTMLElement | null>(null);
 const currentSlide = ref(0);
 const emit = defineEmits(["update:currentSlide"]);
```

Comment:
and in the name of the branch please use the DALA ticket number of the whole ticket, not the subticket

Reply:
I discussed about it with Florian and Emanuel and for this ticket makes more sense to have the sub-tickets as independent task. In case we will convert the sub-tasks in task

Reply:
oh okay, then nevermind!